testnets/_IBC/kimatestnet-osmosistestnet.json

```diff
@@ -0,0 +1,32 @@
+{
+  "$schema": "../ibc_data.schema.json",
+  "chain_1": {
+    "chain_name": "kimatestnet",
```

Comment:
This chain must be registered before IBC connections can be registered. This will fail validation until then